PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -1185,17 +1185,16 @@ - (void)handleLiveFeedEvents:(PNSubscribeStatus *)status {
                 
                 // Check whether event has been triggered on presence channel or channel group.
                 // In case if check will return YES this is presence event.
-                BOOL isPresenceEvent = ([PNChannel isPresenceObject:event[@"actualChannel"]] ||
-                                        [PNChannel isPresenceObject:event[@"subscribedChannel"]]);
+                BOOL isPresenceEvent = (event[@"presenceEvent"] ? YES : NO);
                 if (isPresenceEvent) {
                     
                     if (event[@"subscribedChannel"]) {
                         
-                        event[@"subscribedChannel"] = [PNChannel channelForPresence:event[@"subscribedChannel"]];
+                        event[@"presence"][@"subscribedChannel"] = [PNChannel channelForPresence:event[@"subscribedChannel"]];
```

Comment:
@jzucker2 But why you decided to change location of data which should be at top _event's_ object level?